controllers/comments.controller.js

```diff
@@ -21,3 +22,20 @@ exports.getAllComments = (request, response, next) => {
             next(err);  
         });
 };
+
+exports.postComments = (request, response, next) => {
+    
+    const { author, body } = request.body
+    const { article_id } = request.params
+    if (!author || !body){
+        return response.status(400).send({msg: "Missing input"})
+    }
```

Comment:
You have error handling middleware that is handling this kind of response, so for consistency with your other custom error handling it would be better to have this as a Promise.reject with a `status` and `msg`.